packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -0,0 +1,239 @@
+import '../ui/plotly.dart';
+
+class FramesBarPlotly {
+  static const int CPU_GOOD_TRACE = 0;
+  static const int  CPU_JANK_TRACE = 1;
+  static const int  GPU_GOOD_TRACE = 2;
+  static const int  GPU_JANK_TRACE = 3;
+  static const int  CPU_HIGH_WATER_MARK_TRACE = 4;
+  static const int  GPU_HIGH_WATER_MARK_TRACE = 5;
+
+  static const String CPU_COLOR = 'rgba(100, 181, 246, 1)';    // #64B5F6
+  static const String GPU_COLOR = 'rgba(77, 182, 172, 1)';     // #4DB6AC
+
+  static const String CPU_JANK_COLOR = 'rgba(255, 0, 0, .8)';
+  static const String GPU_JANK_COLOR = 'rgba(255, 0, 0, .7)';
+
+  // Number of barcharts in zoom in view.
```

Comment:
do you mean the number of bar chart bars?

Reply:
Yep. Fixed the comment.